workloadDriver/src/main/java/com/vmware/weathervane/workloadDriver/common/core/loadPath/FindMaxLoadPath.java

```diff
@@ -433,6 +435,11 @@ private UniformLoadInterval nextInterval() {
 				curStatusInterval.setEndUsers(curUsers);
 				curStatusInterval.setDuration(warmupIntervalSec);
 				
+				statsIntervalComplete = true;
+				curStatsInterval.setName(nextInterval.getName());
+				curStatsInterval.setUsers(curUsers);
+				curStatsInterval.setDuration(warmupIntervalSec);
```

Comment:
name and duration set again, as above.

Reply:
Same thing, curStatusInterval Vs curStatsInterval